src/site/includes/metatags.liquid

```diff
@@ -31,8 +31,13 @@
 <meta content="{{ tag }}" property="article:tag">
   {% endfor %}
 {% endif %}
+{% if keywords %}
+<meta name ="keywords" content="{{keywords}}">
+{% endif %}
 {% if title === 'Home' %}
 <title>VA.gov</title>
+{% elseif title === 'Find VA Locations | Veterans Affairs' %}
```

Comment:
Can we just remove this from all but the home page?

Reply:
The template adds ": VA.gov" at the end of every title. It was requested to remove ": VA.gov"  specifically for the Facility Locator.

Reply:
Facility locator is on VA.gov so this makes its title inconsistent with the rest of the website.

Reply:
We're aware. This request is from DSVA and has been discussed in this ticket: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/17409#issuecomment-480314839

Reply:
Might be reading this wrong but [this](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17409#issuecomment-480360100) suggests that removing :VA.gov should be a site wide change.